packages/dashboard-core-plugins/src/linker/LinkerLink.tsx

```diff
@@ -106,22 +250,117 @@ export class LinkerLink extends PureComponent<LinkerLinkProps> {
+      dropdownOffsetX = -25;
+      dropdownOffsetY = 10;
+    }
+
+    let symbol = '';
+    if (operator !== undefined) {
+      if (operator === 'startsWith') {
+        symbol = 'a*';
+      } else if (operator === 'endsWith') {
+        symbol = '*z';
+      } else {
+        symbol = TableUtils.getFilterOperatorString(operator);
+      }
+    }
+
     return (
-      <svg className={className}>
-        <clipPath id={clipPathId}>
-          <path d={selectClipPath} clipRule="evenodd" />
-        </clipPath>
-        <path
-          className="link-select"
-          d={path}
-          onClick={this.handleClick}
-          clipPath={`url(#${clipPathId})`}
-        />
-        <path className="link-background" d={path} />
-        <path className="link-foreground" d={path} />
-        <circle className="link-dot" cx={x1} cy={y1} r="5" />
-        <polygon className="link-triangle" points={points} />
-      </svg>
+      <>
+        <svg
+          className={classNames(className, {
+            hovering: isHovering,
+          })}
+        >
+          <clipPath id={clipPathId}>
+            <path d={selectClipPath} clipRule="evenodd" />
+          </clipPath>
+          <path
+            className="link-select"
+            d={path}
+            onClick={this.handleClick}
+            clipPath={`url(#${clipPathId})`}
+            onMouseEnter={this.handleMouseEnter}
+            onMouseLeave={this.handleMouseLeave}
+          />
+          <path className="link-background" d={path} />
+          <path className="link-foreground" d={path} />
+          <circle className="link-dot" cx={x1} cy={y1} r="5" />
+          <polygon className="link-triangle" points={points} />
+        </svg>
+        {startColumnType != null && isSelected && (
+          <>
+            <Button
+              kind="primary"
+              className="btn-fab btn-operator"
+              style={{
+                top: midY + dropdownOffsetY,
+                left: midX + dropdownOffsetX,
+              }}
+              onClick={() => {
+                // no-op: click is handled in `DropdownMenu'
+              }}
+              icon={
+                <div className="fa-md fa-layers">
+                  <b>{symbol}</b>
+                  <FontAwesomeIcon
+                    icon={vsTriangleDown}
+                    transform="right-8 down-10 shrink-6"
+                  />
+                </div>
+              }
+              tooltip="Change comparison operator"
+            >
+              <DropdownMenu
+                actions={this.getDropdownActions}
+                popperOptions={{ placement: 'bottom-start' }}
+              />
+            </Button>
+
+            <Button
+              kind="primary"
+              className="btn-fab btn-delete"
+              style={{ top: midY + deleteOffsetY, left: midX + deleteOffsetX }}
+              onClick={this.handleDelete}
+              icon={<FontAwesomeIcon icon={vsTrash} transform=" down-1" />}
```

Comment:
```suggestion
              icon={vsTrash} />}
```

Shouldn't need any positional offset with the CSS centering change I shared above.